website/package.json

```diff
@@ -20,6 +20,7 @@
   },
   "dependencies": {
     "@docusaurus/core": "2.0.1",
+    "@docusaurus/plugin-content-blog": "^2.0.1",
```

Comment:
do we need this plugin ?

Reply:
I was planning for posting blog posts later on and hence, decided to keep the blog prereqs, is that fine?

Reply:
I would say we remove it for now as it's not used.
Only when we're using it, we can introduce it